lib/index.js

```diff
@@ -214,7 +214,7 @@ Server.prototype = {
         ((this.startUpConfig.proxiedHost !== undefined) || (this.startUpConfig.proxiedPort !== undefined))) {
       const host = this.startUpConfig.proxiedHost;
       const port = this.startUpConfig.proxiedPort;
-      yield checkProxiedHost(host, port);
+      yield checkProxiedHost(host, port, this.userConfig);
```

Comment:
instead of passing in the whole config object, just pass in the parameter. because then we chance its location over time without effecting the function itself